README.md

```diff
@@ -465,6 +465,102 @@ yarn test $adapter/test/unit/my-specific-test.test.ts
 yarn test --watch $adapter/test/unit
 ```
 
+#### Soak Testing
+
+In order to soak test adapters we need to create and push the adapter out to the sdlc cluster. From there we can use the Flux Emulator or K6 to send traffic to it for the amount of time you need.
+
+Prerequisites to starting an external adapter in the sdlc cluster
+
+1. You must be on the vpn to access the k8s cluster.
+2. You must have your kubeconfig set to the sdlc cluster which also requires you be logged into the aws secure-sdlc account as a user with k8s permissions. To do so it would look something like this but with your specific profile name `aws sso login --profile sdlc`. Instructions to set this up and set your kubeconfig can be found here: https://www.notion.so/chainlink/QA-Kubernetes-Cluster-ca3f1a64e6fd4476ac5a76c8bfcd8624
+3. In order to pull the external adapter helm chart you need to have a GitHub PAT and add the chainlik helm repo using the instructions here: https://github.com/smartcontractkit/charts
+
```

Comment:
I wonder if the README is the best place for this, since it's only helpful for the Chainlink team. Maybe we can think about reorganizing the README in the future...

Reply:
Definitely. I wasn't sure where to put all the info. I can move it out to notion if that makes more sense.

Reply:
I think for now this is fine, and we can move it if it gets too cluttered :)